swift/ql/lib/codeql/swift/generated/ParentChild.qll

```diff
@@ -0,0 +1,2182 @@
+// generated by codegen/codegen.py
+import codeql.swift.elements
+
+module Impl {
```

Comment:
Could this module be `private`?

Reply:
yeah, I wanted to, then noticed I didn't do it and fixed it with [3f4a330](https://github.com/github/codeql/pull/10211/commits/3f4a33065865ea257e845c52d0856a4e25eeea3d) :slightly_smiling_face: